src/MOIWrapper.jl

```diff
@@ -259,13 +260,46 @@ function MOIU.load_variables(optimizer::Optimizer, nvars::Integer)
     b = zeros(m)
     c = zeros(nvars)
     optimizer.data = ModelData(m, nvars, I, J, V, b, 0., c)
+    if length(optimizer.sol.primal) != nvars
```

Comment:
What could the length be if not `nvars`? Would `isempty` also work as a test? Same below.

Reply:
`sol` contains the solution of the previous optimization. If the user has added new variables, then it is not `nvars`

Reply:
Does this interface allow adding variables and/or constraints?

Reply:
You can copy a model with a different number of variables/constraints. Normally, they will arrive in the same order.

Reply:
Could this `if` be replaced with `resize!` then?